packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -310,17 +310,17 @@ class Neo4jGraphQL {
 
         const { validateTypeDefs, validateResolvers } = this.parseStartupValidationConfig();
 
-        if (validateTypeDefs) {
-            validateDocument(document, directives, types);
-        }
+        const baseSchema = validateDocument(document, validateTypeDefs, directives, types);
+        validateCustomResolverRequires(document, baseSchema);
```

Comment:
With my other suggestion, I think this is tidier:

```suggestion
        validateDocument(document, validateTypeDefs, directives, types);
```